client-side.md

```diff
@@ -33,6 +33,13 @@ See the separate [npm-packages.md](./npm-packages.md).
     otherwise.
 -   Prefer locking your dependencies (direct and transitive) using a lockfile rather than committing them into Git.
 
+## Node.js
+
+-   Prefer using the _Current_ even-numbered [version of Node.js](https://nodejs.org/en/about/releases/)
```

Comment:
It would be helpful to include your PR comment "Odd-numbered versions become unsupported more quickly than even-numbered." here to explain why you encourage even-numbered releases